packages/server/src/models/user.entity.ts

```diff
@@ -27,7 +26,7 @@ export default class User {
   public password: string;
 
   @Column()
-  public status: UserStatuses;
+  public status: string;
```

Comment:
Why did you decide to abandon using enums? I'm using them heavily after your hint here :)

Reply:
E.g. as single values and arrays:
https://github.com/dubbha/croton/pull/81/files#diff-d584a668c576ebf201af646c073a277bR8-R14

Reply:
Well, for some reason it is verified as an Object by `typeorm` `@Column` decorator. The error you have sent in Slack has been caused by that, it seems

Reply:
`@Column('simple-enum')` should work.
But you leave it if you don't want to try.